src/swagger/trip.swagger.js

```diff
@@ -357,3 +357,51 @@
  *         description: Trip ID doesn't exists
  *
  */
+
+/**
+ * @swagger
+ * definitions:
+ *   deleteComment:
+ *     type: object
+ *     properties:
+ *       comment:
+ *         type: string
+ *       required:
+ *         - comment
+ */
+
+
+/**
+ * @swagger
+ * /api/trips/requests/{requestId}/comments/{commentId}:
+ *   delete:
+ *     tags:
+ *       - trips
+ *     name: Delete comment
+ *     summary: User should be able to delete comment posted on thread
+ *     produces:
+ *       - application/json
+ *     consumes:
+ *       - application/json
+ *     parameters:
+ *       - in: header
+ *         name: Authorization
+ *         required: true
+ *         type: string
+ *       - name: tripId
+ *         in: path
```

Comment:
Parameters in a path are always `required:true` checkout this guide [https://swagger.io/docs/specification/describing-parameters/](https://swagger.io/docs/specification/describing-parameters/) the section named Required and Optional Parameters